R/bm-tpc-h.R

```diff
@@ -103,12 +103,10 @@ tpc_h <- Benchmark("tpc_h",
       # of the query validation output data when rounded to the nearest 1/100th
       # > For results from SUM aggregates, the resulting values must be within
       # $100 of the query validation output data.
-      all_equal_out <- all.equal(result, answer, tolerance = 0.01)
+      all_equal_out <- waldo::compare(result, answer, tolerance = 0.01)
```

Comment:
`waldo::compare` is totally fine here, but we have check note complaining about not importing anything from {waldo}. We do use `compare` again below, so it's probably easiest to actually import compare with a roxygen tag + get this Note out of the way at the same time 

```
❯ checking dependencies in R code ... NOTE
  Namespace in Imports field not imported from: ‘waldo’
    All declared Imports should be used.
```

Reply:
Huh, that seems like an odd choice in r-devel; `::`-only imports are everywhere. Added a roxygen import but didn't remove the namespacing so it's still easy to see where it comes from.

And yes, I ran tests with the DuckDB tests enabled and it passes; just lots of those deparse warnings.

Reply:
Awesome, thanks!